src/debuginfo.cpp

```diff
@@ -59,7 +59,7 @@ using namespace llvm;
 #include <cstdio>
 #include <cassert>
 
-#if defined(LLVM35) && !defined(LLVM36)
+#if JL_LLVM_VERSION == 30500
```

Comment:
This should in principle still be a range since LLVM 3.5.1 does exist. Not sure if it actually matters though since we might not compile there anyway....